src/components/Session/SessionFilter.vue

```diff
@@ -1,6 +1,9 @@
 <template>
   <article :class="{['session-filter']: true, active: active}">
     <template v-if="!isFilterCollection">
+      <button :class="{fab: true, CalendarMultipleCheck: true, active: active}" @click="checkMySchedule">
+        <icon-mdi-calendar-multiple-check></icon-mdi-calendar-multiple-check>
+      </button>
```

Comment:
```suggestion
    <button class="fab CalendarMultipleCheck" :class="{ active }" @click="checkMySchedule">
      <icon-mdi-calendar-multiple-check />
    </button>
    <template v-if="!isFilterCollection">
```

- 沒有條件的 `class` 可以不用寫在 `v-bind` 裡面
- icon 元件因為沒有內容，可以寫成 self-closing
- 如果這個按鈕不受 `v-if` 的條件影響，直接放在 `<article>` 底下就可以了

Reply:
在 [319f36a](https://github.com/COSCUP/2024/commit/319f36abeb768f25f36e16f453ce48467e215084) 用 ESLint 跑過了，也為按鈕新增文字提示（title 屬性）、調整背景顏色。

Reply:
看起來都沒問題了👍
建議之後的 CSS Class 用比較常見的 kebab-case 命名法喔